lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -122,14 +134,16 @@ export class CryptoOps implements ICrypto {
      * @param kid 
      */
     async removeTokenBindingKey(kid: string): Promise<boolean> {
-        return this.cache.delete(kid);
+        return this.cache.asymmetricKeys.delete(kid);
     }
 
     /**
      * Removes all cryptographic keys from IndexedDB storage
      */
     async clearKeystore(): Promise<boolean> {
-        return this.cache.clear();
+        const dataStoreNames = Object.keys(this.cache);
+        const databaseStorage = this.cache[dataStoreNames[0]];
+        return (databaseStorage) ? await databaseStorage.deleteDatabase() : false;
```

Comment:
Nit: Does this need to be in parens?
```suggestion
        return databaseStorage ? await databaseStorage.deleteDatabase() : false;
```

Reply:
It doesn't, I'm just used to formatting ternaries like that. Without a logical expression there I guess the parentheses are superfluous.